internal/terminal/table_test.go

```diff
@@ -256,3 +260,90 @@ func TestTablePayload(t *testing.T) {
 		assert.Equal(t, data, payloadData[logFieldData])
 	})
 }
+
+func TestParseValue(t *testing.T) {
+
+	for _, tc := range []struct {
+		description    string
+		value          interface{}
+		expectedString string
+	}{
+		{
+			description:    "nil",
+			value:          nil,
+			expectedString: "",
+		},
+		{
+			description:    "empty strings",
+			value:          "",
+			expectedString: "",
+		},
+		{
+			description:    "strings",
+			value:          "string",
+			expectedString: "string",
+		},
+		{
+			description:    "empty generic slices",
+			value:          []interface{}{},
+			expectedString: "[]",
+		},
+		{
+			description:    "empty string slices",
+			value:          []string{},
+			expectedString: "[]",
+		},
+		{
+			description:    "string slices",
+			value:          []string{"slice", "of", "strings"},
+			expectedString: "[slice of strings]",
+		},
+		{
+			description:    "int slices",
+			value:          []int{1, 2, 3},
+			expectedString: "[1 2 3]",
+		},
+		{
+			description:    "generic slices",
+			value:          []interface{}{1, "2", []int{3, 3, 3}},
+			expectedString: "[1 2 [3 3 3]]",
+		},
+		{
+			description:    "integers",
+			value:          42,
+			expectedString: "42",
+		},
+		{
+			description:    "negative integers",
+			value:          -42,
+			expectedString: "-42",
+		},
+		{
+			description:    "whole number floats",
+			value:          42.0,
+			expectedString: "42",
+		},
+		{
+			description:    "floats",
+			value:          42.120,
+			expectedString: "42.12",
+		},
+		{
+			description: "structs",
+			value: struct {
+				foo int
+				bar string
+			}{foo: 42, bar: "bar"},
+			expectedString: "{42 bar}",
+		},
+	} {
+		t.Run("parseValue should correctly parse "+tc.description, func(t *testing.T) {
```

Comment:
[nit] instead of `"...should correctly parse..."` let's have these test descriptions say exactly what the assert is:

`"parseValue should parse 'a nil value as an empty string'"`
`"parseValue should parse 'an empty string as an empty string'"`

I know it seems like overkill, but really helps the reading test output if/when any error occurs.  the same comment applies for the "pointers" test below